package/PartSeg/launcher_main.py

```diff
@@ -46,15 +46,18 @@ def _test_imports():  # pragma: no cover
     w1 = AnalysisMain("test")
     w2 = MaskMain("test")
     w3 = MainWindow("test")
-    console = QtConsole(napari.Viewer())
+    v = napari.Viewer()
```

Comment:
**suggestion:** Consider using a context manager for viewer lifecycle management

While the current approach of explicitly closing the viewer is good, using a `with` statement could provide more robust resource management, especially if this code were to be used in a production environment.